lib/paper_trail/version_concern.rb

```diff
@@ -210,9 +212,8 @@ def reify(options = {})
     # not have an `object_changes` text column.
     def changeset
       return nil unless self.class.column_names.include? 'object_changes'
-
-      _changes = self.class.object_changes_col_is_json? ? object_changes : PaperTrail.serializer.load(object_changes)
-      @changeset ||= HashWithIndifferentAccess.new(_changes).tap do |changes|
+      _changes = HashWithIndifferentAccess.new(object_changes_deserialized)
+      @changeset ||= _changes.tap do |changes|
         if PaperTrail.serialized_attributes?
           item_type.constantize.unserialize_attribute_changes_for_paper_trail!(changes)
         end
```

Comment:
Since this is purely a code style PR, I'm chiming in here to say that IMO `tap` is an anti-pattern because it adds one more thing to understand and read without adding much funcationality. I'd personally prefer this code style:
```ruby
@changeset ||= begin
  changes = HashWithIndifferentAccess.new(object_changes_deserialized)
  if PaperTrail.serialized_attributes?
    item_type.constantize.unserialize_attribute_changes_for_paper_trail!(changes)
  end
  changes
end
```

It also has the performance improvement of only having to run `HashWithIndifferentAccess.new(object_changes_deserialized)` if `changeset` hasn't been called before, whereas currently it will be run every time `changeset` is called.